Makefile

```
@@ -127,6 +129,7 @@ E2E_CONF_FILE ?= $(ROOT_DIR)/test/e2e/config/azure-dev.yaml
 E2E_CONF_FILE_ENVSUBST := $(ROOT_DIR)/test/e2e/config/azure-dev-envsubst.yaml
 SKIP_CLEANUP ?= false
 SKIP_CREATE_MGMT_CLUSTER ?= false
+WIN_REPO_LIST := https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/images/image-repo-list
```

We should provide the option to overwrite this URL so we can easily switch to image-repo-list-master by defining an environment variable:

```suggestion
WIN_REPO_LIST ?= https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/images/image-repo-list
```

---

good catch, that was the intent